packages/common/src/ui/layout/skeletons/ButtonSkeleton.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Skeleton } from '@mui/material';
+
+export const ButtonSkeleton: React.FC = () => {
+  return (
+    <Skeleton
+      variant="rectangular"
+      height={40}
+      width={125}
+      style={{ borderRadius: 24, transform: 'scale(1,1)' }}
```

Comment:
by default the skeletons scale to `(1, 0.6)` in order to give some spacing. it means that the buttons aren't the right height, and I don't need it to scale / space here